File: features/discovery/autocomplete/src/main/java/com/tokopedia/autocomplete/adapter/decorater/SpacingItemDecoration.java
package com.tokopedia.autocomplete.adapter.decorater;

import android.graphics.Rect;
import androidx.recyclerview.widget.RecyclerView;
import android.view.View;

public class SpacingItemDecoration extends RecyclerView.ItemDecoration {

    private final int dimen;

    public SpacingItemDecoration(int dimen) {
        this.dimen = dimen;
    }

    public void getItemOffsets(Rect outRect, View view, RecyclerView parent, RecyclerView.State state) {
        outRect.right = this.dimen;
        outRect.top = this.dimen / 2;
        outRect.bottom = this.dimen / 2;
    }
}

File: features/merchant/product_manage_list/src/main/java/com/tokopedia/product/manage/feature/filter/presentation/adapter/decorator/SpacingItemDecoration.kt
package com.tokopedia.product.manage.feature.filter.presentation.adapter.decorator

import android.graphics.Rect
import android.view.View
import androidx.recyclerview.widget.RecyclerView

class SpacingItemDecoration(private val dimen: Int) : RecyclerView.ItemDecoration() {

    override fun getItemOffsets(outRect: Rect, view: View, parent: RecyclerView, state: RecyclerView.State) {
        outRect.right = this.dimen
        outRect.top = this.dimen / 2
        outRect.bottom = this.dimen / 2
    }
}
